packages/bindgen/spec.yml

```diff
@@ -667,6 +667,8 @@ classes:
       read_group: () -> Group&
       duplicate: () -> TransactionRef
 
+      update_schema: '(schema: std::vector<ObjectSchema>, version: uint64_t, migration_function: Nullable<std::function<(old_realm: SharedRealm, new_realm: SharedRealm, new_schema_handle: Schema&) -> void>>, initialization_function: Nullable<std::function<(realm: SharedRealm) -> void>>, in_transaction: bool) -> void'
```

Comment:
I don't think the generated C++ would like that .. cc @RedBeard0531?

Reply:
I played around with this just now and it does seem like this would not work. So I will just leave it this way, if this ends up being jarring we could introduce type aliases but it's not that much longer than other functions like this.